Portal/Services/WasteService.cs

```diff
@@ -25,7 +25,8 @@ public async Task<DuringWhichMonthRequestViewModel> GetCurrentQuarter(int journe
             var duringWhichMonthRequestViewModel = new DuringWhichMonthRequestViewModel
             {
                 JourneyId = journeyId,
-                WasteType = await _httpWasteService.GetWasteType(journeyId)
+                // We're not part of a journey yet, so this can't really be hooked up
```

Comment:
Does this mean that we're going to have to hard code the waste type in our views for the time being?